### PR TITLE
:arrow_up::one::two: web_debranding

### PR DESCRIPTION
**Pull request by rbikbov**

### CODE REVIEW

**Review by ufaks — APPROVED** [review]

File: web_debranding_support/views.xml [hunk 1]
```diff
@@ -1,13 +0,0 @@
...
-    <template id="menu_secondary" inherit_id="web.menu_secondary" priority="32">
-        <xpath expr="//div[@class='oe_footer']" position="replace">
-            <div class="oe_footer">
-                <span>Supported by <a target="_new" t-att-href="request.env['ir.config_parameter'].get_param('support_branding.company_url')" t-att-style="'color: ' + request.env['ir.config_parameter'].get_param('support_branding.company_color')"><t t-esc="request.env['ir.config_parameter'].get_param('support_branding.company_name')" /></a></span>
-                <div t-if="request.env['ir.config_parameter'].get_param('support_branding.release')">Version <t t-esc="request.env['ir.config_parameter'].get_param('support_branding.release')" /></div>
-            </div>
-        </xpath>
-    </template>
-
-</data></openerp>
```
Comment (yelizariev) [inline web_debranding_support/views.xml:13]: It's empty module now. So, just remove it completly

**Review by yelizariev — CHANGES_REQUESTED** [review]
Also, readme probably contains wrong statement:

 1. Deletes Odoo label in footer, i.e.
 
     Powered by Odoo

File: web_debranding/README.rst [hunk 2]
```diff
@@ -50,9 +47,14 @@ Removes references to odoo.com:
 18. Hides Enterprise features in Settings
 19. Replaces "Odoo" in all backend qweb templates
 
-    This provides a big part of debranding. You can find examples at menu ``[[ Invoicing ]] >> Dashboard >> Reconcile 5 items >> 'Automatic reconcilation' button title on hover``
+    This provides a big part of debranding. You can find examples in any tree view if you click ``Import`` button, then paste next code in browser javascript console:
+     ``$('.oe_import_noheaders.text-muted').text()``, e.g. at menu ``[[ Settings ]] >> Users & Companies >> Users >> Import``
```
Comment (yelizariev) [inline web_debranding/README.rst]: Prepare js code that makes that hint visible.

 Also move *e.g. at menu ...* to brackets after *click ``Import`` button*

By the way, for button you have to use special format https://odoo-development.readthedocs.io/en/latest/dev/docs/rst-requirements.html#buttons

File: web_debranding/README.rst [hunk 3]
```diff
@@ -50,9 +47,14 @@ Removes references to odoo.com:
...
+
+      If the file contains
+            the column names, **Odoo** can try auto-detecting the
```
Comment (yelizariev) [inline web_debranding/README.rst]: Is there special reason why those lines have different indent?

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: web_debranding/README.rst (hunk 2 above)
Comment (rbikbov) [reply web_debranding/README.rst]: Done.

File: web_debranding/README.rst (hunk 3 above)
Comment (rbikbov) [reply web_debranding/README.rst]: No, I just saved the format that the user sees. Fixed.

**Review by yelizariev — APPROVED** [review]
Previous remarks are done :+1:

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
1. The first message from OdooBot (not from Bot)
2. 'OdooBot' is not replaced to 'Bot' in the user profile section Preferences>Messaging and Social. 
3. Odoo social links on the main page and on the contact page.
4. If you create a new record on the Translated Terms (Settings->Translations), selecting the value 'Model Field' for Type field will result in an error.
5. 'Odoo Server error' is displayed in the first error message above.

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Pull request merged**